app/brave_generated_resources.grd

```diff
@@ -216,27 +216,27 @@
       </message>
 
       <!-- Widevine -->
-      <message name="IDS_WIDEVINE_PERMISSION_REQUEST_TEXT_FRAGMENT" desc="Text fragment for Widevine permission request. 'Widevine' is the name of a plugin and should not be translated.">
+      <message name="IDS_WIDEVINE_PERMISSION_REQUEST_TEXT_FRAGMENT" desc="Text fragment for Widevine permission request. 'Widevine' is the name of a plugin and should not be translated">
```

Comment:
`desc` element is not shown in the UI and is intended for translators. There's no need to remove periods inside the `desc` attributes.